app/liquid_tags/github_tag.rb

```diff
@@ -1,5 +1,5 @@
 class GithubTag < LiquidTagBase
-  REGISTRY_REGEXP = %r{https://github\.com/[\w\-.]{1,39}/[\w\-.]{1,39}/?((issues|pull)/\d+((#issuecomment-|#discussion_|#pullrequestreview-)\w+)?)?}
+  REGISTRY_REGEXP = %r{https://github\.com/[\w\-.]{1,39}/[\w\-.]{1,39}/?((issues|pull)/\d+((#issuecomment-|#discussion_|#pullrequestreview-)\w+)?)?(\sno-readme\$|\snoreadme\$)?}
```

Comment:
I think `(\sno-readme\$|\snoreadme\$)?` could be compressed to `(\sno-?readme\$)?`. The `-?` means to treat the `-` as optional (as in 0 or 1 `-` characters).